src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+server.servlet.contextPath=/api/v1
+springdoc.api-docs.path=/api-docs
```

Comment:
What do you think about rather using /api/docs. More common standard.

We could even do /api/v1/docs but that would imply that all the docs are v1 docs, so /api/docs is probs the best.